src/network/bridge.rs

```diff
@@ -558,6 +577,7 @@ fn create_veth_pair<'fd>(
     netns: &mut netlink::Socket,
     data: &InternalData,
     primary_index: u32,
+    bridge_mac: Option<Vec<u8>>,
```

Comment:
These arguments are too much as per clippy. Probably its time to move all these args to a struct.

Reply:
added `#[allow(clippy::too_many_arguments)]` I rather not rework most of this code here right now considering that we likely need to backport this.